### PR TITLE
Operator v1.6

### PR DESCRIPTION
**Pull request by Nidhi251289**
We will be releasing this today.

### CODE REVIEW

File: post-install-script [hunk 1]
```diff
@@ -0,0 +1,1690 @@
+#!/bin/bash
```
Comment (zlogic) [inline post-install-script:1]: I think this file was added here by accident.

Comment (Nidhi251289) [reply post-install-script]: fixed.

**Review by MarkvanMents — APPROVED** [review]
Hi @Nidhi251289 ,
I've made a few minor changes. Is it OK to publish?
Thanks for making the PR.

**Comment by Nidhi251289** [comment]
Looks good to me. Kindly proceed ahead with publishing.

**Pull request merged**